spec/models/recipe_ingredients_spec.rb

```diff
@@ -6,4 +6,17 @@
     it { should belong_to :recipe }
     it { should belong_to :ingredient }
   end
+
+  describe "class method" do
+    before(:each) do
+      @rec1 = Recipe.create!(name: "Tacos", complexity: 5, genre: "Mexican")
+      @ing1 = Ingredient.create!(name: "Ground Beef", cost: 2)
+      @ing2 = Ingredient.create!(name: "Salt", cost: 4)
+      @recing1 = RecipeIngredient.create!(recipe_id: @rec1.id, ingredient_id: @ing1.id)
+      @recing1 = RecipeIngredient.create!(recipe_id: @rec1.id, ingredient_id: @ing2.id)
+    end
+    it "returns total cost of ingredients" do
+      expect(RecipeIngredient.total_cost).to eq(6)
+    end
```

Comment:
I think this is a false positive on the method you wrote. If we only want the total cost of all ingredients for a specific recipe, this will pass because we only have one recipe. If we create another recipe with more ingredients, I think this would fail and would've flagged that the query wasn't quite right.